test/functional/dataElements/xdmObject/helpers/platformMocks.js

```diff
@@ -11,8 +11,10 @@ const SCHEMAS_ENDPOINT_REGEX = /\/schemaregistry\/tenant\/schemas(\?|$)/;
 const SCHEMA_ENDPOINT_REGEX = /\/schemaregistry\/tenant\/schemas\/.+/;
 const IDENTITIES_ENDPOINT_REGEX = /\/data\/core\/idnamespace\/identities(\?|$)/;
 
+const HTTP_METHOD_GET = "GET";
+
 export const sandboxesUnauthorized = RequestMock()
-  .onRequestTo(SANDBOXES_ENDPOINT_REGEX)
+  .onRequestTo({ url: SANDBOXES_ENDPOINT_REGEX, method: HTTP_METHOD_GET })
```

Comment:
Without specifying the GET method, this mock was _also_ mocking the related CORS OPTIONS request, which was causing a failedToFetch issue because, in this case, we were returning a 401 status code for the OPTIONS request rather than just the GET.